examples/linear_model/plot_bayesian_ridge_curvefit.py

```diff
@@ -28,33 +28,36 @@
 
 # Author: Yoshihiro Uchida <nimbus1after2a1sun7shower@gmail.com>
 
+# %%
+# Generate sinusoidal data with noise
+# -----------------------------------
 import numpy as np
 import matplotlib.pyplot as plt
 
-from sklearn.linear_model import BayesianRidge
-
 
 def func(x):
     return np.sin(2 * np.pi * x)
 
 
-# #############################################################################
-# Generate sinusoidal data with noise
 size = 25
 rng = np.random.RandomState(1234)
 x_train = rng.uniform(0.0, 1.0, size)
 y_train = func(x_train) + rng.normal(scale=0.1, size=size)
 x_test = np.linspace(0.0, 1.0, 100)
 
 
-# #############################################################################
+# %%
 # Fit by cubic polynomial
+# -----------------------
 n_order = 3
 X_train = np.vander(x_train, n_order + 1, increasing=True)
 X_test = np.vander(x_test, n_order + 1, increasing=True)
 
-# #############################################################################
+# %%
 # Plot the true and predicted curves with log marginal likelihood (L)
+# -------------------------------------------------------------------
+from sklearn.linear_model import BayesianRidge
+
 reg = BayesianRidge(tol=1e-6, fit_intercept=False, compute_score=True)
```

Comment:
Based on your PR comment, I suspect you meant to move these lines into the previous fitting block rather than this plotting one. I agree that the fitting block would be a better place for these lines given that this final block is only concerned with plotting.

Reply:
Done.

Thanks for taking the time to review!